daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -31,33 +31,43 @@ import Test.Main
 import Test.Tasty
 import Test.Tasty.HUnit
 import qualified Web.JWT as JWT
+import qualified Data.ByteString.Lazy as BSL
+import qualified Data.HashMap.Strict as HMS
+import Data.Aeson
 
+import DA.Directory
 import DA.Bazel.Runfiles
 import DA.Daml.Helper.Run
 import SdkVersion
 
 main :: IO ()
-main =
-    withTempDir $ \tmpDir -> do
+main = do
+    setEnv "TASTY_NUM_THREADS" "1" True
     -- We manipulate global state via the working directory and
     -- the environment so running tests in parallel will cause trouble.
-    setEnv "TASTY_NUM_THREADS" "1" True
+    yarn : damlTypesPath : args <- getArgs
```

Comment:
Unfortunately, doing this is the only way I've found to reliably get at the directory containing yarn.

Reply:
That's fine. We're doing this in other places too.